src/css/components/_layout.scss

```diff
@@ -86,7 +86,7 @@ body.vjs-full-window {
   /* Fix for IE6 full-window. http://www.cssplay.co.uk/layouts/fixed.html */
   overflow-y: auto;
 }
-.video-js.vjs-fullscreen {
+.vjs-full-window .video-js.vjs-fullscreen {
```

Comment:
Do the top/bottom/left/right need to be here or can they stay in the non-full-window vjs-fullscreen?